science/iverilog/Portfile

```diff
@@ -1,20 +1,21 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+name                iverilog
```

Comment:
This explicit `name` line can be removed, because `github.setup` sets the `name` implicitly.